Code/wwlib/chunkio.h

```diff
@@ -150,7 +163,7 @@ class ChunkSaveClass
 	bool					End_Micro_Chunk();
 
 	// Write data into the file
-	uint32				Write(const void *buf, uint32 nbytes);
```

Comment:
I think this is going to cause a problem, specifically with the read/write.  `size_t` is different on 32-bit vs. 64-bit systems

Reply:
Only if `nbytes >= 0xffffffff`, but then you have bigger issues.